model_compression_toolkit/qat/keras/quantizer/quantization_dispatcher_builder.py

```diff
@@ -47,10 +55,16 @@ def quantization_dispatcher_builder(n: common.BaseNode,
     if n.is_weights_quantization_enabled():
         attributes = fw_info.get_kernel_op_attributes(n.type)
         for attr in attributes:
-            qunatizer_class = method2quantizer.get(n.final_weights_quantization_cfg.weights_quantization_method)
+            qunatizer_class = method2weightquantizer.get(n.final_weights_quantization_cfg.weights_quantization_method)
             if qunatizer_class is None:
                 common.Logger.error(
                     f'Unknown Quantiztion method: {n.final_weights_quantization_cfg.weights_quantization_method}')
             nqd.add_weight_quantizer(attr, qunatizer_class(n.final_weights_quantization_cfg))
 
+    if n.is_activation_quantization_enabled():
+        # single output -> normalize to list of output_shapes
+        output_shapes = n.output_shape if isinstance(n.output_shape[0], (list, tuple)) else [n.output_shape]
+        qunatizer_class = method2actquantizer.get(n.final_activation_quantization_cfg.activation_quantization_method)
+        nqd.activation_quantizers = [qunatizer_class(n.final_activation_quantization_cfg)] * len(output_shapes)
```

Comment:
Is it correct for reused layers? What if its outputs should be quantized differently?

Reply:
Even for a reused layers, the quantization must be the same.